packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -0,0 +1,224 @@
+/**
+ * Copyright 2020, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Bin, CirclePlus, PenStroke, ThumbUp, Zap } from '@sumup/icons';
+import { fireEvent } from '@testing-library/dom';
+import React, { useRef, useState } from 'react';
+
+import {
+  create,
+  renderToHtml,
+  axe,
+  RenderFn,
+  render,
+  act,
+  userEvent,
+} from '../../util/test-utils';
+import Button from '../Button';
+
+import { PopoverItem, PopoverItemProps, Popover } from './Popover';
+
+const placements = ['auto', 'top', 'bottom', 'left', 'right'];
+
+describe('PopoverItem', () => {
+  function renderPopoverItem<T>(
+    renderFn: RenderFn<T>,
+    props: PopoverItemProps,
+  ) {
+    return renderFn(<PopoverItem {...props} />);
+  }
+
+  const baseProps = { children: 'PopoverItem' };
+
+  describe('styles', () => {
+    it('should render as Link when an href (and onClick) is passed', () => {
```

Comment:
Should `onClick` still be called in this case, or just filtered out? If yes, maybe we should also assert it in another test

(I assume it should, this could contain things like sending an event)